app/src/androidTest/java/com/wire/android/utils/DeepLinkProcessorTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.wire.android.utils
+
+import android.net.Uri
+import com.wire.android.util.EMPTY
+import com.wire.android.util.deeplink.DeepLinkProcessor
+import com.wire.android.util.deeplink.DeepLinkResult
+import io.mockk.coEvery
+import io.mockk.impl.annotations.MockK
+import org.amshove.kluent.internal.assertEquals
+import org.junit.jupiter.api.Assertions.assertInstanceOf
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+
+class DeepLinkProcessorTest {
+    private var deepLinkProcessor = DeepLinkProcessor()
+
+    @Test
+    fun test1() {
```

Comment:
We follow the pattern given..when..then for naming testing methods, let's rename it 🙂

Reply:
This whole file was not meant to be here. it was by mistake, the updated one just replaced. Thanks